projects/js-packages/number-formatters/composer.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "automattic/js-number-formatters",
+	"description": "Number formatting utilities.",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"scripts": {
+		"build-production": [
+			"pnpm run build"
+		],
+		"build-development": [
+			"pnpm run build"
+		],
+		"test-js": [
+			"pnpm run test"
+		],
+		"test-coverage": [
+			"pnpm run test-coverage"
+		]
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"extra": {
+		"autotagger": true,
+		"npmjs-autopublish": true,
+		"mirror-repo": "Automattic/number-formatters",
```

Comment:
@anomiex  Is this sound? You suggested `Automattic/number-formatter` (singular) in your code, which makes me wonder if that would be a better name overall. I've used "number-formatters" plural throughout 🤔

Reply:
Either is fine as far as I'm concerned. 🙂

Reply:
Plural to match the npm package name is probably the best way to go. :+1: